docs/src/paradox/03-webapi/api-v2/getting-lists.md

```diff
@@ -0,0 +1,43 @@
+<!---
+Copyright © 2015-2018 the contributors (see Contributors.md).
+
+This file is part of Knora.
+
+Knora is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Knora is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+# Gettings Lists
```

Comment:
"Gettings" -> "Getting"

Reply:
done in https://github.com/dhlab-basel/Knora/pull/852/commits/22deb149333baf81097fd3a1dc49d02d40d0e8b6